EIPS/eip-6882.md

```diff
@@ -0,0 +1,112 @@
+     *   wants to receive it on his own wallet, or a different address if the beneficiary is a
+     *   different wallet
+     * @return The final amount withdrawn
+     */
+    function withdraw(address asset, uint256 amount, address receiver) external returns (uint256);
+
+    /**
+     * @notice Allows users to borrow a specific `amount` of the reserve underlying asset, provided that the borrower
+     * already supplied enough collateral, or he was given enough allowance by a credit delegator on the
+     * corresponding debt token
+     * - E.g. User borrows 100 USDC passing as `receiver` his own address, receiving the 100 USDC in his wallet
+     *   and 100 debt tokens
+     * @param asset The address of the underlying asset to borrow
+     * @param amount The amount to be borrowed
+     * @param receiver The address of the user who will receive the debt. Should be the address of the borrower itself
+     * calling the function if he wants to borrow against his own collateral, or the address of the credit delegator
+     * if he has been given credit delegation allowance
+     */
+    function borrow(address asset, uint256 amount, address receiver, address owner) external returns (uint256);
+
+    /**
+     * @notice Repays a borrowed `amount` on a specific reserve, burning the equivalent debt tokens owned
+     * @param asset The address of the borrowed underlying asset previously borrowed
+     * @param amount The amount to repay
+     * @param onBehalfOf The address of the user who will get his debt reduced/removed. Should be the address of the
+     * user calling the function if he wants to reduce/remove his own debt, or the address of any other
+     * other borrower whose debt should be removed
+     * @return The final amount repaid
+     */
+    function repay(address asset, uint256 amount, address onBehalfOf) external returns (uint256);
+
+    /*
+     * Allow third party accounts to borrow tokens on behalf of the owner.
+     * @param delegatee The address of the third party account to allow
+     * @param allowed True if the delegatee is allowed to borrow tokens on behalf of the owner, false otherwise
+     */
+    function allow(address delegatee, bool allowed) external;
+
+    /*
+     * Returns true if the delegatee is allowed to borrow tokens on behalf of the owner.
+     * @param owner The address of the owner
+     * @param account The address of the delegatee
+     */
+    function isAllowed(address owner, address account) external view returns (bool);
+}
+
+```
+
+## Rationale
+
+The rationale behind creating a standard API for borrowing tokens is to improve the user and developer experience when interacting with lending protocols. By providing a borrowing aggregator, users can access aggregated data from multiple lending protocols in a single place and easily move funds between them to achieve the best borrowing rates.
```

Comment:
This paragraph belongs in the Motivation section. The Rationale section should justify individual technical choices made within the proposal itself (eg. why `withdraw` returns the final amount withdrawn), while the Motivation section justifies the proposal as a whole.